[2024-07-12T01:47:49.544+0530] {processor.py:161} INFO - Started process (PID=36667) to work on //home/kali/Desktop/projects/git/bank_data_processing/dags/dag_raw_recurring_deposits.py
[2024-07-12T01:47:49.545+0530] {processor.py:830} INFO - Processing file //home/kali/Desktop/projects/git/bank_data_processing/dags/dag_raw_recurring_deposits.py for tasks to queue
[2024-07-12T01:47:49.546+0530] {logging_mixin.py:188} INFO - [2024-07-12T01:47:49.546+0530] {dagbag.py:545} INFO - Filling up the DagBag from //home/kali/Desktop/projects/git/bank_data_processing/dags/dag_raw_recurring_deposits.py
[2024-07-12T01:47:49.566+0530] {logging_mixin.py:188} INFO - [2024-07-12T01:47:49.565+0530] {dagbag.py:355} ERROR - Failed to import: //home/kali/Desktop/projects/git/bank_data_processing/dags/dag_raw_recurring_deposits.py
Traceback (most recent call last):
  File "/home/kali/Desktop/projects/git/bank_data_processing/myenv/lib/python3.11/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "//home/kali/Desktop/projects/git/bank_data_processing/dags/dag_raw_recurring_deposits.py", line 13, in <module>
    from scripts.raw_load_recurring_deposits import create_tables,tuncate_table,insert_table
ModuleNotFoundError: No module named 'scripts'
[2024-07-12T01:47:49.567+0530] {processor.py:842} WARNING - No viable dags retrieved from //home/kali/Desktop/projects/git/bank_data_processing/dags/dag_raw_recurring_deposits.py
[2024-07-12T01:47:49.591+0530] {processor.py:183} INFO - Processing //home/kali/Desktop/projects/git/bank_data_processing/dags/dag_raw_recurring_deposits.py took 0.053 seconds
[2024-07-12T01:48:19.806+0530] {processor.py:161} INFO - Started process (PID=37129) to work on //home/kali/Desktop/projects/git/bank_data_processing/dags/dag_raw_recurring_deposits.py
[2024-07-12T01:48:19.807+0530] {processor.py:830} INFO - Processing file //home/kali/Desktop/projects/git/bank_data_processing/dags/dag_raw_recurring_deposits.py for tasks to queue
[2024-07-12T01:48:19.810+0530] {logging_mixin.py:188} INFO - [2024-07-12T01:48:19.809+0530] {dagbag.py:545} INFO - Filling up the DagBag from //home/kali/Desktop/projects/git/bank_data_processing/dags/dag_raw_recurring_deposits.py
[2024-07-12T01:48:19.828+0530] {logging_mixin.py:188} INFO - [2024-07-12T01:48:19.827+0530] {dagbag.py:355} ERROR - Failed to import: //home/kali/Desktop/projects/git/bank_data_processing/dags/dag_raw_recurring_deposits.py
Traceback (most recent call last):
  File "/home/kali/Desktop/projects/git/bank_data_processing/myenv/lib/python3.11/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "//home/kali/Desktop/projects/git/bank_data_processing/dags/dag_raw_recurring_deposits.py", line 13, in <module>
    from scripts.raw_load_recurring_deposits import create_tables,tuncate_table,insert_table
ModuleNotFoundError: No module named 'scripts'
[2024-07-12T01:48:19.828+0530] {processor.py:842} WARNING - No viable dags retrieved from //home/kali/Desktop/projects/git/bank_data_processing/dags/dag_raw_recurring_deposits.py
[2024-07-12T01:48:19.846+0530] {processor.py:183} INFO - Processing //home/kali/Desktop/projects/git/bank_data_processing/dags/dag_raw_recurring_deposits.py took 0.049 seconds
[2024-07-12T01:48:51.460+0530] {processor.py:161} INFO - Started process (PID=37520) to work on //home/kali/Desktop/projects/git/bank_data_processing/dags/dag_raw_recurring_deposits.py
[2024-07-12T01:48:51.461+0530] {processor.py:830} INFO - Processing file //home/kali/Desktop/projects/git/bank_data_processing/dags/dag_raw_recurring_deposits.py for tasks to queue
[2024-07-12T01:48:51.463+0530] {logging_mixin.py:188} INFO - [2024-07-12T01:48:51.463+0530] {dagbag.py:545} INFO - Filling up the DagBag from //home/kali/Desktop/projects/git/bank_data_processing/dags/dag_raw_recurring_deposits.py
[2024-07-12T01:48:51.480+0530] {logging_mixin.py:188} INFO - [2024-07-12T01:48:51.479+0530] {dagbag.py:355} ERROR - Failed to import: //home/kali/Desktop/projects/git/bank_data_processing/dags/dag_raw_recurring_deposits.py
Traceback (most recent call last):
  File "/home/kali/Desktop/projects/git/bank_data_processing/myenv/lib/python3.11/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "//home/kali/Desktop/projects/git/bank_data_processing/dags/dag_raw_recurring_deposits.py", line 13, in <module>
    from scripts.raw_load_recurring_deposits import create_tables,tuncate_table,insert_table
ModuleNotFoundError: No module named 'scripts'
[2024-07-12T01:48:51.480+0530] {processor.py:842} WARNING - No viable dags retrieved from //home/kali/Desktop/projects/git/bank_data_processing/dags/dag_raw_recurring_deposits.py
[2024-07-12T01:48:51.498+0530] {processor.py:183} INFO - Processing //home/kali/Desktop/projects/git/bank_data_processing/dags/dag_raw_recurring_deposits.py took 0.045 seconds
[2024-07-12T01:48:56.271+0530] {processor.py:161} INFO - Started process (PID=37575) to work on //home/kali/Desktop/projects/git/bank_data_processing/dags/dag_raw_recurring_deposits.py
[2024-07-12T01:48:56.272+0530] {processor.py:830} INFO - Processing file //home/kali/Desktop/projects/git/bank_data_processing/dags/dag_raw_recurring_deposits.py for tasks to queue
[2024-07-12T01:48:56.276+0530] {logging_mixin.py:188} INFO - [2024-07-12T01:48:56.275+0530] {dagbag.py:545} INFO - Filling up the DagBag from //home/kali/Desktop/projects/git/bank_data_processing/dags/dag_raw_recurring_deposits.py
[2024-07-12T01:48:56.299+0530] {logging_mixin.py:188} INFO - [2024-07-12T01:48:56.298+0530] {dagbag.py:355} ERROR - Failed to import: //home/kali/Desktop/projects/git/bank_data_processing/dags/dag_raw_recurring_deposits.py
Traceback (most recent call last):
  File "/home/kali/Desktop/projects/git/bank_data_processing/myenv/lib/python3.11/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "//home/kali/Desktop/projects/git/bank_data_processing/dags/dag_raw_recurring_deposits.py", line 13, in <module>
    from models.raw_models.raw_load_recurring_deposits import create_tables,tuncate_table,insert_table
ModuleNotFoundError: No module named 'models'
[2024-07-12T01:48:56.300+0530] {processor.py:842} WARNING - No viable dags retrieved from //home/kali/Desktop/projects/git/bank_data_processing/dags/dag_raw_recurring_deposits.py
[2024-07-12T01:48:56.332+0530] {processor.py:183} INFO - Processing //home/kali/Desktop/projects/git/bank_data_processing/dags/dag_raw_recurring_deposits.py took 0.071 seconds
[2024-07-12T01:49:26.647+0530] {processor.py:161} INFO - Started process (PID=38026) to work on //home/kali/Desktop/projects/git/bank_data_processing/dags/dag_raw_recurring_deposits.py
[2024-07-12T01:49:26.648+0530] {processor.py:830} INFO - Processing file //home/kali/Desktop/projects/git/bank_data_processing/dags/dag_raw_recurring_deposits.py for tasks to queue
[2024-07-12T01:49:26.651+0530] {logging_mixin.py:188} INFO - [2024-07-12T01:49:26.651+0530] {dagbag.py:545} INFO - Filling up the DagBag from //home/kali/Desktop/projects/git/bank_data_processing/dags/dag_raw_recurring_deposits.py
[2024-07-12T01:49:26.670+0530] {logging_mixin.py:188} INFO - [2024-07-12T01:49:26.669+0530] {dagbag.py:355} ERROR - Failed to import: //home/kali/Desktop/projects/git/bank_data_processing/dags/dag_raw_recurring_deposits.py
Traceback (most recent call last):
  File "/home/kali/Desktop/projects/git/bank_data_processing/myenv/lib/python3.11/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "//home/kali/Desktop/projects/git/bank_data_processing/dags/dag_raw_recurring_deposits.py", line 13, in <module>
    from models.raw_models.raw_load_recurring_deposits import create_tables,tuncate_table,insert_table
ModuleNotFoundError: No module named 'models'
[2024-07-12T01:49:26.670+0530] {processor.py:842} WARNING - No viable dags retrieved from //home/kali/Desktop/projects/git/bank_data_processing/dags/dag_raw_recurring_deposits.py
[2024-07-12T01:49:26.689+0530] {processor.py:183} INFO - Processing //home/kali/Desktop/projects/git/bank_data_processing/dags/dag_raw_recurring_deposits.py took 0.049 seconds
[2024-07-12T01:49:57.603+0530] {processor.py:161} INFO - Started process (PID=38354) to work on //home/kali/Desktop/projects/git/bank_data_processing/dags/dag_raw_recurring_deposits.py
[2024-07-12T01:49:57.604+0530] {processor.py:830} INFO - Processing file //home/kali/Desktop/projects/git/bank_data_processing/dags/dag_raw_recurring_deposits.py for tasks to queue
[2024-07-12T01:49:57.605+0530] {logging_mixin.py:188} INFO - [2024-07-12T01:49:57.605+0530] {dagbag.py:545} INFO - Filling up the DagBag from //home/kali/Desktop/projects/git/bank_data_processing/dags/dag_raw_recurring_deposits.py
[2024-07-12T01:49:57.623+0530] {logging_mixin.py:188} INFO - [2024-07-12T01:49:57.621+0530] {dagbag.py:355} ERROR - Failed to import: //home/kali/Desktop/projects/git/bank_data_processing/dags/dag_raw_recurring_deposits.py
Traceback (most recent call last):
  File "/home/kali/Desktop/projects/git/bank_data_processing/myenv/lib/python3.11/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "//home/kali/Desktop/projects/git/bank_data_processing/dags/dag_raw_recurring_deposits.py", line 13, in <module>
    from models.raw_models.raw_load_recurring_deposits import create_tables,tuncate_table,insert_table
ModuleNotFoundError: No module named 'models'
[2024-07-12T01:49:57.623+0530] {processor.py:842} WARNING - No viable dags retrieved from //home/kali/Desktop/projects/git/bank_data_processing/dags/dag_raw_recurring_deposits.py
[2024-07-12T01:49:57.643+0530] {processor.py:183} INFO - Processing //home/kali/Desktop/projects/git/bank_data_processing/dags/dag_raw_recurring_deposits.py took 0.046 seconds
[2024-07-12T01:50:28.312+0530] {processor.py:161} INFO - Started process (PID=38771) to work on //home/kali/Desktop/projects/git/bank_data_processing/dags/dag_raw_recurring_deposits.py
[2024-07-12T01:50:28.313+0530] {processor.py:830} INFO - Processing file //home/kali/Desktop/projects/git/bank_data_processing/dags/dag_raw_recurring_deposits.py for tasks to queue
[2024-07-12T01:50:28.314+0530] {logging_mixin.py:188} INFO - [2024-07-12T01:50:28.314+0530] {dagbag.py:545} INFO - Filling up the DagBag from //home/kali/Desktop/projects/git/bank_data_processing/dags/dag_raw_recurring_deposits.py
[2024-07-12T01:50:28.336+0530] {logging_mixin.py:188} INFO - [2024-07-12T01:50:28.334+0530] {dagbag.py:355} ERROR - Failed to import: //home/kali/Desktop/projects/git/bank_data_processing/dags/dag_raw_recurring_deposits.py
Traceback (most recent call last):
  File "/home/kali/Desktop/projects/git/bank_data_processing/myenv/lib/python3.11/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "//home/kali/Desktop/projects/git/bank_data_processing/dags/dag_raw_recurring_deposits.py", line 13, in <module>
    from models.raw_models.raw_load_recurring_deposits import create_tables,tuncate_table,insert_table
  File "//home/kali/Desktop/projects/git/bank_data_processing/dags/models/raw_models/raw_load_recurring_deposits.py", line 6, in <module>
    import pandas as pd
ModuleNotFoundError: No module named 'pandas'
[2024-07-12T01:50:28.336+0530] {processor.py:842} WARNING - No viable dags retrieved from //home/kali/Desktop/projects/git/bank_data_processing/dags/dag_raw_recurring_deposits.py
[2024-07-12T01:50:28.356+0530] {processor.py:183} INFO - Processing //home/kali/Desktop/projects/git/bank_data_processing/dags/dag_raw_recurring_deposits.py took 0.050 seconds
[2024-07-12T01:50:58.656+0530] {processor.py:161} INFO - Started process (PID=39203) to work on //home/kali/Desktop/projects/git/bank_data_processing/dags/dag_raw_recurring_deposits.py
[2024-07-12T01:50:58.657+0530] {processor.py:830} INFO - Processing file //home/kali/Desktop/projects/git/bank_data_processing/dags/dag_raw_recurring_deposits.py for tasks to queue
[2024-07-12T01:50:58.661+0530] {logging_mixin.py:188} INFO - [2024-07-12T01:50:58.660+0530] {dagbag.py:545} INFO - Filling up the DagBag from //home/kali/Desktop/projects/git/bank_data_processing/dags/dag_raw_recurring_deposits.py
[2024-07-12T01:50:58.687+0530] {logging_mixin.py:188} INFO - [2024-07-12T01:50:58.685+0530] {dagbag.py:355} ERROR - Failed to import: //home/kali/Desktop/projects/git/bank_data_processing/dags/dag_raw_recurring_deposits.py
Traceback (most recent call last):
  File "/home/kali/Desktop/projects/git/bank_data_processing/myenv/lib/python3.11/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "//home/kali/Desktop/projects/git/bank_data_processing/dags/dag_raw_recurring_deposits.py", line 13, in <module>
    from models.raw_models.raw_load_recurring_deposits import create_tables,tuncate_table,insert_table
  File "//home/kali/Desktop/projects/git/bank_data_processing/dags/models/raw_models/raw_load_recurring_deposits.py", line 6, in <module>
    import pandas as pd
ModuleNotFoundError: No module named 'pandas'
[2024-07-12T01:50:58.687+0530] {processor.py:842} WARNING - No viable dags retrieved from //home/kali/Desktop/projects/git/bank_data_processing/dags/dag_raw_recurring_deposits.py
[2024-07-12T01:50:58.718+0530] {processor.py:183} INFO - Processing //home/kali/Desktop/projects/git/bank_data_processing/dags/dag_raw_recurring_deposits.py took 0.071 seconds
[2024-07-12T01:51:29.726+0530] {processor.py:161} INFO - Started process (PID=39621) to work on //home/kali/Desktop/projects/git/bank_data_processing/dags/dag_raw_recurring_deposits.py
[2024-07-12T01:51:29.727+0530] {processor.py:830} INFO - Processing file //home/kali/Desktop/projects/git/bank_data_processing/dags/dag_raw_recurring_deposits.py for tasks to queue
[2024-07-12T01:51:29.729+0530] {logging_mixin.py:188} INFO - [2024-07-12T01:51:29.728+0530] {dagbag.py:545} INFO - Filling up the DagBag from //home/kali/Desktop/projects/git/bank_data_processing/dags/dag_raw_recurring_deposits.py
[2024-07-12T01:51:29.768+0530] {logging_mixin.py:188} INFO - [2024-07-12T01:51:29.765+0530] {dagbag.py:355} ERROR - Failed to import: //home/kali/Desktop/projects/git/bank_data_processing/dags/dag_raw_recurring_deposits.py
Traceback (most recent call last):
  File "/home/kali/Desktop/projects/git/bank_data_processing/myenv/lib/python3.11/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "//home/kali/Desktop/projects/git/bank_data_processing/dags/dag_raw_recurring_deposits.py", line 13, in <module>
    from models.raw_models.raw_load_recurring_deposits import create_tables,tuncate_table,insert_table
  File "//home/kali/Desktop/projects/git/bank_data_processing/dags/models/raw_models/raw_load_recurring_deposits.py", line 6, in <module>
    import pandas as pd
ModuleNotFoundError: No module named 'pandas'
[2024-07-12T01:51:29.769+0530] {processor.py:842} WARNING - No viable dags retrieved from //home/kali/Desktop/projects/git/bank_data_processing/dags/dag_raw_recurring_deposits.py
[2024-07-12T01:51:29.798+0530] {processor.py:183} INFO - Processing //home/kali/Desktop/projects/git/bank_data_processing/dags/dag_raw_recurring_deposits.py took 0.079 seconds
[2024-07-12T01:52:06.768+0530] {processor.py:161} INFO - Started process (PID=40426) to work on //home/kali/Desktop/projects/git/bank_data_processing/dags/dag_raw_recurring_deposits.py
[2024-07-12T01:52:06.771+0530] {processor.py:830} INFO - Processing file //home/kali/Desktop/projects/git/bank_data_processing/dags/dag_raw_recurring_deposits.py for tasks to queue
[2024-07-12T01:52:06.775+0530] {logging_mixin.py:188} INFO - [2024-07-12T01:52:06.774+0530] {dagbag.py:545} INFO - Filling up the DagBag from //home/kali/Desktop/projects/git/bank_data_processing/dags/dag_raw_recurring_deposits.py
[2024-07-12T01:52:07.424+0530] {logging_mixin.py:188} INFO - [2024-07-12T01:52:07.423+0530] {base.py:84} INFO - Using connection ID 'mysql_default' for task execution.
[2024-07-12T01:52:07.434+0530] {processor.py:840} INFO - DAG(s) 'raw_load_recurring_deposits' retrieved from //home/kali/Desktop/projects/git/bank_data_processing/dags/dag_raw_recurring_deposits.py
[2024-07-12T01:52:07.611+0530] {logging_mixin.py:188} INFO - [2024-07-12T01:52:07.609+0530] {override.py:1847} INFO - Created Permission View: can delete on DAG:raw_load_recurring_deposits
[2024-07-12T01:52:07.635+0530] {logging_mixin.py:188} INFO - [2024-07-12T01:52:07.634+0530] {override.py:1847} INFO - Created Permission View: can read on DAG:raw_load_recurring_deposits
[2024-07-12T01:52:07.649+0530] {logging_mixin.py:188} INFO - [2024-07-12T01:52:07.648+0530] {override.py:1847} INFO - Created Permission View: can edit on DAG:raw_load_recurring_deposits
[2024-07-12T01:52:07.650+0530] {logging_mixin.py:188} INFO - [2024-07-12T01:52:07.649+0530] {dag.py:3096} INFO - Sync 1 DAGs
[2024-07-12T01:52:07.673+0530] {logging_mixin.py:188} INFO - [2024-07-12T01:52:07.673+0530] {dag.py:3118} INFO - Creating ORM DAG for raw_load_recurring_deposits
[2024-07-12T01:52:07.675+0530] {logging_mixin.py:188} INFO - [2024-07-12T01:52:07.674+0530] {dag.py:3954} INFO - Setting next_dagrun for raw_load_recurring_deposits to None, run_after=None
[2024-07-12T01:52:07.715+0530] {processor.py:183} INFO - Processing //home/kali/Desktop/projects/git/bank_data_processing/dags/dag_raw_recurring_deposits.py took 0.966 seconds
